tests/typegen/evaluation/test_metric_data_calculator.py

```diff
@@ -10,7 +10,7 @@ def get_sample_data():
         "sample_original_filename",
         None,
         "sample_function_name",
-        10,
+        0,
```

Comment:
Why the change? Is this a relative line number offset from the original function declaration?

Reply:
Instead of using the line number, we now use the column offset. However, even without the column offset, we can still determine the parameter as there can not be another parameter with the same name. Additionally, keeping the column offset causes the following issues:
```Python
def f(a: int, b: bool):
    pass

def f(a: float, b: bool):
    pass
```
The column offsets of the 2 b parameters are different and we do not gain anything by keeping the values. Thus, we can set it to 0.

Reply:
Do we gain anything by using the column data then?

Reply:
It is useful for local variables. For example:

```Python
e: int = 1
if True:
    e: str = "string"
```
We can differentiate between the first e and the second e. I have already added this case in the metric data calculator test.